crates/ironrdp-connector/src/connection.rs

```diff
@@ -977,8 +978,8 @@ fn create_client_confirm_active(
             keyboard_ime_filename: config.ime_file_name.clone(),
         }),
         CapabilitySet::Pointer(Pointer {
-            color_pointer_cache_size: 0,
-            pointer_cache_size: 0,
+            color_pointer_cache_size: DEFAULT_POINTER_CACHE_SIZE,
+            pointer_cache_size: DEFAULT_POINTER_CACHE_SIZE,
```

Comment:
Nit: maybe add a comment above this explaining how this is really important (as you said on slack)